src/panels/config/backup/components/ha-backup-upload.ts

```diff
@@ -0,0 +1,60 @@
+import { css, html, LitElement, nothing } from "lit";
+import { customElement, property } from "lit/decorators";
+import "../../../../components/ha-alert";
+import "../../../../components/ha-file-upload";
+import { mdiFolderUpload } from "@mdi/js";
+import type { HomeAssistant } from "../../../../types";
+import type { LocalizeFunc } from "../../../../common/translations/localize";
+
+export const SUPPORTED_FORMAT = "application/x-tar";
+
+export interface BackupFileFormData {
+  file?: File;
+}
+
+export const INITIAL_FORM_DATA: BackupFileFormData = {
+  file: undefined,
+};
+
+@customElement("ha-backup-upload")
```

Comment:
What is the use of this component if it just a wrapper for `ha-file-upload` without extra functionality?